dashboard.csss-ci.com/server/api/controllers/rest/v1/Proposals.php

```diff
@@ -150,6 +175,9 @@ public function visit_put() {
         }
     }
 
+    /**
+     * Inserts certification ifnormation into the proposals model database if all necessary fields are valid, otherwise throws an error  
```

Comment:
spelling

This method is also updating, not inserting. It updates whether or not a certain proposal has receiver certification or not